lib/local-links-manager/import/csv_downloader.rb

```diff
@@ -26,4 +33,10 @@ def downloaded_csv
 
     response.body
   end
+
+  def name_converter(conversions)
+    lambda do |field|
+      conversions.detect { |k, _| k == field } ? conversions[field] : field
```

Comment:
We can simplify this to `conversions.fetch(field, field)` which I admit is a little opaque.  Maybe `conversions.key?(field) ? conversions[field] : field` would be less opaque while still a bit simpler than using `detect`.